enterprise/backend/test/metabase_enterprise/serialization/v2/e2e/yaml_test.clj

```diff
@@ -168,13 +174,23 @@
           (testing "for metrics"
             (is (= 30 (count (dir->file-set (io/file dump-dir "Metric")))))
             (doseq [{:keys [entity_id name] :as metric} (get entities "Metric")
-                    :let [filename (str entity_id "+" name ".yaml")]]
+                    :let [filename (str entity_id "+" (#'u.yaml/truncate-label name) ".yaml")]]
               (is (= (-> metric
                          (dissoc :serdes/meta)
                          (update :created_at u.date/format)
                          (update :updated_at u.date/format))
                      (yaml/from-file (io/file dump-dir "Metric" filename))))))
 
+          (testing "for native query snippets"
+            (is (= 10 (count (dir->file-set (io/file dump-dir "NativeQuerySnippet")))))
+            (doseq [{:keys [entity_id name] :as snippet} (get entities "NativeQuerySnippet")
+                    :let [filename (str entity_id "+" (#'u.yaml/truncate-label name) ".yaml")]]
```

Comment:
It seems a little fragile to be duplicating the filename generation logic that presumably exists somewhere else here in the tests. If we decide to change that code, all these tests will be broken until they're updated as well. Or if there's a bug in the filename generation code this test will start failing which could lead you to believe there's an issue with native query snippets rather than filename generation. Can you just call the existing filename generation code? If not, could it be refactored a bit so you can use it here and don't have to duplicate the logic anywhere else?